css/tabstrip.css

```diff
@@ -56,8 +55,10 @@
   transform: scale(0.8);
   transition: transform 150ms ease;
   width: 100px;
+  overflow: hidden;
```

Comment:
This is not supposed to be required. The image is supposed to be the exact size of its container.